data-files/test/experimentconfig.Any

```diff
@@ -0,0 +1,93 @@
+{
```

Comment:
These appear to be in the `test` directory inside `data-files`. I'd prefer for them to be somewhere else, though it's probably not a big problem for them to be there since they could be used as reference configs in a pinch. I'll resolve this conversation since this location is better then being in the root of `data-files`.